cmake/Findnodejs.cmake

```diff
@@ -16,7 +16,7 @@ find_path(NODEJS_DEV_INCLUDE_DIR node_lib.h PATHS
 "${CMAKE_SOURCE_DIR}/externals/node-v9.0.0/src"
  NO_DEFAULT_PATH
 )
-find_library(NODEJS_LIBRARY NAMES node node.lib libnode.so.59 libnode.59.dylib
+find_library(NODEJS_LIBRARY NAMES node node.lib libnode.so.59 libnode.59.dylib libnode.so.61 libnode.61.dylib
```

Comment:
This is weird... shouldn't we be very sure of which version of Node we're using? Or is using a wildmark (or this solution) a good idea?

Reply:
In the end this line should contain the first node version with our lib interface. 

Until then we can just add all versions that could match during development, as kind of a workaround. There should always be only one binary of libnode in our development environments anyway.